### PR TITLE
Add eksa namespace when getting vspheremachinetemplate

### PR DESCRIPTION
**Pull request by jiayiwang7**
*Issue #, if available:*

to fix failing flux e2e tests; fix typo.

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/framework/flux.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func (e *E2ETest) ValidateFlux() {
 	}
 
 	if err = e.validateWorkerNodeReplicaUpdates(ctx); err != nil {
-		e.T.Errorf("Error validting scaling of Flux managed cluster: %v", err)
+		e.T.Errorf("Error validating scaling of Flux managed cluster: %v", err)
```
Comment (danbudris) [inline test/framework/flux.go:96]: you have become the master 🥷

**Review by danbudris — APPROVED** [review]

**Comment by jaxesn** [comment]
Is this a bug fix for flux support itself?

**Comment by jiayiwang7** [comment]
> Is this a bug fix for flux support itself?

no, just for e2e testing

**Review by g-gaston — APPROVED** [review]

**Pull request merged**